web/modules/custom/foreningsmentor/templates/signup-email-body.html.twig

```diff
@@ -1,26 +1,26 @@
 <h1>Ny tilmelding</h1>
 <p>Der er kommet en ny tilmelding på {{ params.site_name }}.</p>
 <h2>Barnets information</h2>
-<h4>Barnets navn</h4>
+<h3>Navn</h3>
 <p>{{ params.form_values.child_name }}</p>
-<h4>Barnets fødselsdag</h4>
+<h3>Fødselsdag</h3>
 <p>{{ params.form_values.birth_date }}</p>
-<h4>Barnets køn</h4>
+<h3>Køn</h3>
 <p>{{ params.form_values.sex }}</p>
-<h4>Barnets skole</h4>
+<h3>Skole</h3>
 <p>{{ params.form_values.school }}</p>
-<h4>Barnets klassetrin</h4>
+<h3>Klassetrin</h3>
 <p>{{ params.form_values.grade }}</p>
-<h2>Forældre information</h2>
-<h4>Navn</h4>
+<h2>Forældreinformation</h2>
```

Comment:
This should be “Forælderinformation” (there's only one parent in this case), cf. https://dsn.dk/ordbog/ro/foraelder/.

Reply:
Changed